readme.md

```diff
@@ -198,7 +198,13 @@ Refer to the [caching strategies](#caching-strategy) section for more informatio
 
 ### mem.decorator(options)
 
-Returns a TypeScript decorator which memoizes the given function.
+Returns a [decorator](https://github.com/tc39/proposal-decorators) to memoize class methods.
+
+Notes:
+
+- Only class methods and getters/setters can be memoized, not regular functions (they aren't part of the proposal);
```

Comment:
?

```suggestion
- Only class methods and accessors can be memoized, not regular functions (they aren't part of the proposal);
```

Reply:
```suggestion
- Only class methods and accessors can be memoized, not regular functions ([they aren't part of the proposal]());
```

Reply:
I personally prefer `getters/setters`

Reply:
`accessors` is the correct term, but I think `getters/setters` is better as it's more commonly known.